stanford_news.install

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @file
+ */
+
+
+/**
+ * Enable new Stanford News Administration module.
+ */
+function stanford_news_update_7300($sandbox) {
+    if (!module_exists("stanford_news_administration")) {
```

Comment:
@boznik, Do you think it is important that we check for a modified view first? Ie: the view is in the DB? I actually think NOT checking for it is the way we want to go. That should keep it consistent with how its relationship with features is currently. Thoughts?